DonationTest.py

```diff
@@ -31,38 +34,64 @@ def tearDown(self):
         self.driver.quit()
 
     def test_USA_donation_form(self):
-        driver = self.driver
-        params = "?group=test&" + customFunc.test_url_params()
-        page_url = "https://animals-now.org/en/donate/" + params
-        driver.get(page_url)
-        submit_btn = driver.find_element_by_id('fl-main-content').find_element_by_id('paypal-button')
-        submit_btn.click()
-        WebDriverWait(driver, 3).until(EC.new_window_is_opened)
-        driver.switch_to_window(driver.window_handles[1])
-        WebDriverWait(driver, 1).until(EC.url_contains("https://www.paypal.com/donate"))
+        failure_counter = 0
+        failure_status = True
+
+        while failure_counter < TRIES_BEFORE_FAILURE and failure_status:
+            try:
+                driver = self.driver
+                params = "?group=test&" + customFunc.test_url_params()
+                page_url = "https://animals-now.org/en/donate/" + params
+                driver.get(page_url)
+                submit_btn = driver.find_element_by_id('fl-main-content').find_element_by_id('paypal-button')
+                submit_btn.click()
+                WebDriverWait(driver, 3).until(EC.new_window_is_opened)
+                driver.switch_to_window(driver.window_handles[1])
+                WebDriverWait(driver, 1).until(EC.url_contains("https://www.paypal.com/donate"))
+                failure_status = False
```

Comment:
instead you could do that:
```suggestion
                break
```
no?
I don't think you need `failure_status`

Reply:
Yes I see.